obs-websocket-dotnet/Types/OBSScene.cs

```diff
@@ -2,33 +2,41 @@
 using Newtonsoft.Json.Linq;
 using System.Collections.Generic;
 
-namespace OBSWebsocketDotNet.Types
-{
+namespace OBSWebsocketDotNet.Types {
     /// <summary>
     /// Describes a scene in OBS, along with its items
     /// </summary>
-    public class OBSScene
-    {
+    public class OBSScene {
         /// <summary>
         /// OBS Scene name
         /// </summary>
         [JsonProperty(PropertyName = "sceneName")]
         public string Name;
+        
+        /// <summary>
+        /// Scene name as used by GetCurrentProgramScene
+        /// </summary>
+        [JsonProperty(PropertyName = "currentProgramSceneName")]
+        private string ProgramName {
```

Comment:
Syntax should be: 
`public string ProgramName { get; set; }`

Reply:
I would differ. Generally yes, but as an object, `ObsScene` should only have one single Property to expose the scene name. The multiple properties are just a byproduct of different requests providing the scene name in different json properties and is just a "workaround" for multiple json properties mapping to the same .net property. Needs some testing with the SetCurrentProgramScene / SetCurrentPreviewScene though.

If I would change this as proposed, the scene name could be stored in three different properties and you'd never know which one is the right one.